index.html

```diff
@@ -1,72 +1,71 @@
-<!DOCTYPE html>
 <html>
   <head>
     <title>Okteto</title>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0"/>
     <style>
-      html, body { 
+      html, body {{ 
```

Comment:
Why do you have double curly braces?

Reply:
It is to scape it and render it properly

Reply:
If we go with HTML, I'd put the CSS in a different file, so the template here is just about the "business logic"